package.json

```diff
@@ -105,7 +105,6 @@
     "@babel/plugin-proposal-decorators": "^7.23.6",
     "@babel/preset-env": "^7.23.6",
     "@ember/optional-features": "^2.0.0",
-    "@ember/string": "^3.1.1",
```

Comment:
This will still need to be a devDep too right, so we're installing it ourselves when running the test app etc?

Reply:
Absolutely, my bad. Fixed!